src/LearningObjects/LearningObjectInteractor.ts

```diff
@@ -206,6 +206,24 @@ export async function getLearningObjectById(
   }
 }
 
+/**
+ * Fetches a learning objects children by ID
+ *
+ * @export
+ * @param {DataStore} dataStore
+ * @param {string} id the learning object's id
+ */
+export async function getLearningObjectChildrenById(
+  dataStore: DataStore,
+  ObjectId: string,
+){
+  try {
+    return await dataStore.loadChildObjects({id: ObjectId, full: true, accessUnreleased: true});
+  } catch (e) {
+    return Promise.reject(`Problem fetching Children. ${e}`);
```

Comment:
We want to reject error objects instead of strings. We created an object called `LearningObjectError` that defines the errors that learning object service can throw. Try to implement this object when returning errors. 
```suggestion
    reportError(e);
    return Promise.reject(new Error(LearningObjectError.RESOURCE_NOT_FOUND());
```
The reportError function will hand the error object to Sentry. We then return an expected error to the route handler.